src/format/parse.rs

```diff
@@ -561,6 +561,9 @@ impl str::FromStr for DateTime<FixedOffset> {
 
 #[cfg(test)]
 mod tests {
+    use crate::format::InternalInternal as II;
```

Comment:
I think we should probably just import the `InternalInternal::*` variants too, as well as the `Fixed::*` ones. Are these imports in the smallest enclosing scope? It looks like they're all being used in `test_parse()` only.

Reply:
We can't import the variants of both `Fixed` and `Numeric`, because some of the names overlap.

Reply:
Okay, that's unfortunate.